Fabric.Authorization.AccessControl/src/app/services/name-display.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { IFabricPrincipal } from '../models/fabricPrincipal.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NameDisplayService {
+  constructor() { }
+
+  getPrincipalNameToDisplay(principal: IFabricPrincipal): string {
+    if (principal.principalType.toLowerCase() === 'user') {
```

Comment:
I know we use `user` and `group` constants somewhere else to represent `type`. Perhaps we factor those out into a separate constants file/class?